runtime/src/main/java/org/corfudb/runtime/exceptions/SystemUnavailableException.java

```diff
@@ -0,0 +1,13 @@
+package org.corfudb.runtime.exceptions;
+
+import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
+
+/**
+ * Created by rmichoud on 10/31/17.
+ */
+public class SystemUnavailableException extends UnrecoverableCorfuError {
```

Comment:
SystemUnavailableException -> SystemUnavailableError. Also you can move it to the Unrecoverable package.